src/CoreBundle/private/modules/CMSModuleChooser/CMSModuleChooser.class.php

```diff
@@ -1035,9 +1035,8 @@ protected function TriggerWorkflowLockForPage($sPageId = false)
         if (false === $sPageId) {
             $sPageId = $this->global->GetUserData('pagedef');
         }
-        $oCmsTplPage = TCMSActivePage::GetInstance();
```

Comment:
TCMSActivePage could now be deprecated completely?

Reply:
There are still methods in TCMSActivePage that aren't deprecated yet. I'd like to do this in a future version.